examples/mongodbatlas_advanced_cluster/asymmetric-sharded-cluster/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_providers {
+    mongodbatlas = {
+      source  = "mongodb/mongodbatlas"
+      version = "~> 1.17"
```

Comment:
this will have to be updated to 1.18.0 once released ,right?

Reply:
Yes good catch, will include as part of CLOUDP-266096